### ISSUE TITLE
No typedefinition for withCv() and toCvRect() functions

### ISSUE THREAD

**Issue by ehassaan** [justadudewhohacks/face-recognition.js#78]
I was trying to use face-recognition with opencv4nodejs in typescript but i couldn't import using withCv() function as there are no type definition for withCv() function

**Comment by justadudewhohacks** [comment]
Yeah, typings for the opencv4nodejs interface still have to be added
